src/routes/wishlistRoutes.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const Wishlist = require('./Wishlist');
+const Album = require('./Album');
```

Comment:
You may need to rebase on `main` and make sure you have these two models. And then make sure this `require` is using the correct path to use these models